doc/nrf/gs_installing.rst

```diff
@@ -448,6 +448,15 @@ To set up the toolchain, complete the following steps:
           Define the environment variables in the :file:`~/.zephyrrc` file as described in :ref:`build_environment_cli`.
           This lets you avoid setting them every time you open a terminal window.
 
+.. _installing_vsc:
+
+.. rst-class:: numbered-step
+
+Install the Visual Studio Code extension
+****************************************
+
+|vsc_extension_instructions|
+For detailed instructions see the `nRF Connect for Visual Studio Code`_ README file.
```

Comment:
Given the numbered step here, this suggests you need both VSC and SES-NE to work with the NCS. But I think this isn't the case, no, also given the change from `must` to `can` below?

So maybe this suggestion would work -- *note it exceeds the selected lines!*
```suggestion

.. _installing_vsc:
.. _installing_ses:

.. rst-class:: numbered-step

Install the Visual Studio Code extension or the |SES| Nordic Edition
*******************************************************************

You can install either the Visual Studio Code extension or the |SES| (SES) Nordic Edition to open and compile projects in the |NCS|:

* |vsc_extension_instructions|
  For detailed instructions see the `nRF Connect for Visual Studio Code`_ README file.
* SES is free of charge for use with Nordic Semiconductor devices.
   To install the Nordic Edition, complete the following steps:
```
(The two tags at the top are not a typo.)

Reply:
Good point. I decided to add a numbered section called "Install a build IDE" to keep the title short and then nested the two options under that. Does that look better?